client/src/components/RelatedItemsAndOutfit/RelatedProducts/RelatedProducts.jsx

```diff
@@ -24,17 +27,19 @@ const RelatedProducts = ({ prodsInfo, prodsStyles, prodsMeta, currProd }) => {
         }}
         modalRelProd={modalRelProd}
       />
-      <h2 className="related-products-header">RELATED PRODUCTS</h2>
+      <h2 className="related-products-header" id="ratings-reviews-title">RELATED PRODUCTS</h2>
```

Comment:
taken by Ya already, plus i'm seeing 2x of these IDs here - one for related products and one for user outfit